src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/MacPackagingPipeline.java

```diff
@@ -308,18 +358,17 @@ private static void writeAppInfoPlist(
                 .saveToFile(infoPlistFile);
     }
 
-    private static void sign(AppImageBuildEnv<MacApplication, MacApplicationLayout> env) throws IOException {
-
-        final var app = env.app();
+    private static void sign(AppImageBuildEnv<MacApplication, AppImageLayout> env) throws IOException {
 
         final var codesignConfigBuilder = CodesignConfig.build();
-        app.signingConfig().ifPresent(codesignConfigBuilder::from);
```

Comment:
Changes to the `sign()` method look redundant to me. There are no functional changes, just reformatting and replacing `app` with `env.app()`.